apps/feedahelp/src/components/ui/features/errors/Forbidden.test.tsx

```diff
@@ -8,7 +8,7 @@ describe('Forbidden Component', () => {
     const { getByText, getByAltText } = render(<Forbidden />);
     
     const title = getByText("No permission");
-    const message = getByText(/The page you are trying access has restricted access.*Please Login First./);
+    getByText(/The page you are trying access has restricted access.*Please Login First./);
```

Comment:
u sure about this, does this work. it seem how are you using this? you are not keeoing it in temp variables anymore?

Reply:
this line is not needed because this line just gets the text and does not check anything so we can remove all this line or we can add expect(removedVariable).toBeInTheDocument();

Reply:
then delete it